### PR TITLE
PWX-29565 set default scc priority to nil to unblock ocp 4.12 upgrade…

### PR DESCRIPTION
**Pull request by haibinxie**
…, add annotation to set priority

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: drivers/storage/portworx/testspec/portworxSCC.yaml [hunk 1]
```diff
@@ -17,7 +17,6 @@ kind: SecurityContextConstraints
 metadata:
   name: portworx
   resourceVersion: "1"
-priority: 2
```
Comment (pure-jliao) [inline drivers/storage/portworx/testspec/portworxSCC.yaml:20]: the `-` is missing

**Review by pure-jliao — APPROVED** [review]
LGTM after addressing the comments, and can pass CI

File: drivers/storage/portworx/testspec/portworxSCC.yaml (hunk 1 above)
Comment (haibinxie) [reply drivers/storage/portworx/testspec/portworxSCC.yaml:20]: what do you mean? - means this line is removed

Comment (pure-jliao) [reply drivers/storage/portworx/testspec/portworxSCC.yaml:20]: Ohh nvm I thought it's a yaml list...

**Pull request merged**